src/main/kotlin/org/rust/cargo/project/configurable/RustfmtConfigurable.kt

```diff
@@ -5,13 +5,76 @@
 
 package org.rust.cargo.project.configurable
 
+import com.intellij.execution.configuration.EnvironmentVariablesComponent
+import com.intellij.openapi.options.BoundConfigurable
+import com.intellij.openapi.options.ConfigurationException
 import com.intellij.openapi.project.Project
+import com.intellij.openapi.ui.ComboBox
 import com.intellij.openapi.ui.DialogPanel
-import com.intellij.ui.layout.panel
+import com.intellij.ui.RawCommandLineEditor
+import com.intellij.ui.components.Label
+import com.intellij.ui.layout.*
+import org.rust.cargo.project.settings.rustfmtSettings
+import org.rust.cargo.toolchain.RustChannel
+import javax.swing.JComponent
+
+class RustfmtConfigurable(private val project: Project) : BoundConfigurable("Rustfmt") {
+    private val state = project.rustfmtSettings.settingsState
+
+    private val additionalArguments = RawCommandLineEditor()
+
+    private val channelLabel = Label("C&hannel:")
+    private val channel = ComboBox<RustChannel>().apply {
+        RustChannel.values()
+            .sortedBy { it.index }
+            .forEach { addItem(it) }
+    }
+
+    private val environmentVariables = EnvironmentVariablesComponent()
 
-class RustfmtConfigurable(project: Project) : RsConfigurableBase(project, "Rustfmt") {
     override fun createPanel(): DialogPanel = panel {
+        blockRow {
+            labeledRow("&Additional arguments:", additionalArguments) {
+                additionalArguments(pushX, growX)
+                    .comment("Additional arguments to pass to <b>rustfmt</b> or <b>cargo fmt</b> command")
+                    .withBinding(
+                        componentGet = { it.text },
+                        componentSet = { component, value -> component.text = value },
+                        modelBinding = state::additionalArguments.toBinding()
+                    )
+
+                channelLabel.labelFor = channel
+                channelLabel()
+                channel().withBinding(
+                    componentGet = { it.item },
+                    componentSet = { component, value -> component.item = value },
+                    modelBinding = state::channel.toBinding()
+                )
+            }
+
+            row(environmentVariables.label) {
```

Comment:
I don't know why, but the plugin keeps putting this field above the "Additional arguments" field.

<img width="1094" alt="Screen Shot 2021-12-23 at 09 56 11" src="https://user-images.githubusercontent.com/6079006/147202297-1dea74cd-34d0-405e-9e20-512f20c6653c.png">